client/state/data-layer/wpcom/me/settings/profile-links/index.js

```diff
@@ -0,0 +1,178 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import { noop } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import { dispatchRequestEx } from 'state/data-layer/wpcom-http/utils';
+import { http } from 'state/data-layer/wpcom-http/actions';
+import {
+	USER_PROFILE_LINKS_ADD,
+	USER_PROFILE_LINKS_DELETE,
+	USER_PROFILE_LINKS_REQUEST,
+} from 'state/action-types';
+import {
+	addUserProfileLinksDuplicate,
+	addUserProfileLinksError,
+	addUserProfileLinksMalformed,
+	addUserProfileLinksSuccess,
+	deleteUserProfileLinkError,
+	deleteUserProfileLinkSuccess,
+	receiveUserProfileLinks,
+} from 'state/profile-links/actions';
+
+/**
+ * Dispatches a request to fetch profile links of the current user
+ *
+ * @param   {Function} dispatch Redux dispatcher
+ * @param   {Object}   action   Redux action
+ * @returns {Object} Dispatched http action
+ */
+export const requestUserProfileLinks = ( { dispatch }, action ) =>
+	dispatch(
+		http(
+			{
+				apiVersion: '1.1',
+				method: 'GET',
+				path: '/me/settings/profile-links',
+			},
+			action
+		)
+	);
+
+/**
+ * Dispatches a user profile links receive action when the request succeeded.
+ *
+ * @param   {Function} dispatch Redux dispatcher
+ * @param   {Object}   action   Redux action
+ * @param   {Array}    data     Response from the endpoint
+ * @returns {Object} Dispatched user profile links receive action
+ */
+export const handleRequestSuccess = ( { dispatch }, action, { profile_links } ) =>
+	dispatch( receiveUserProfileLinks( profile_links ) );
+
+/**
+ * Dispatches a request to add profile links for the current user
+ *
+ * @param   {Function} dispatch Redux dispatcher
+ * @param   {Object}   action   Redux action
+ * @returns {Object} Dispatched http action
+ */
+export const addUserProfileLinks = ( { dispatch }, action ) =>
+	dispatch(
+		http(
+			{
+				apiVersion: '1.2',
+				method: 'POST',
+				path: '/me/settings/profile-links/new',
```

Comment:
The idea is that this dispatch would go in a separate module at `/me/settings/profile-links/new/index.js`, see this [example](https://github.com/Automattic/wp-calypso/tree/master/client/state/data-layer/wpcom/activity-log/rewind).

Reply:
Ah, makes sense - applied in bacd559af4acca4e4de1493bc0dec858b3d4b54c - I love that structure, thanks! ❤️